docs/GetStarted/getting_started_macos_bazel.md

```diff
@@ -0,0 +1,124 @@
+# Getting Started on macOS with Bazel
+
+<!--
+Notes to those updating this guide:
+
+    * This document should be __simple__ and cover essential items only.
+      Notes for optional components should go in separate files.
+
+    * This document parallels getting_started_linux_bazel.md and
+      getting_started_windows_bazel.md
+      Please keep them in sync.
+-->
+
+This guide walks through building the core compiler and runtime parts of IREE
+from source. Auxiliary components like the Python bindings and Vulkan driver are
+not documented for macOS at this time.
+
+IREE is not officially supported on macOS at this time. It may work, but it is
+not a part of our open source CI, and may be intermittently broken.
+Contributions related to macOS support and documentation are welcome however.
+
+## Prerequisites
+
+### Install Bazel
+
+Install Bazel version > 2.0.0 (see 
+[`.bazelversion`](https://github.com/google/iree/blob/master/.bazelversion) 
+for the specific version IREE uses) by following the 
+[official docs](https://docs.bazel.build/versions/master/install-os-x.html#install-with-installer-mac-os-x).
```

Comment:
Thanks :)

(I like how this is written now, just sharing my experience)
I gave the CMake and Bazel docs a try on my mac where I already had homebrew and Xcode (app) installed. I had basically no issues following these steps. After updating Bazel to the latest version via `brew upgrade bazel`, it prompted me to install the `.bazelversion` version by printing out a shell command to run. I think that's self-explanatory and we don't need to explicitly require people to install the exact version.

Reply:
So something like this?

### Install Homebrew

This guide uses [Homebrew](https://brew.sh/) to install IREE's dependencies.

```shell
$ /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install.sh)"
```

### Install Bazel

Install Bazel via Homebrew:

```shell
$ brew install bazel
```

Note: when you first run `bazel` to build IREE, it will prompt you to copy and run a shell command to select the right version.

Reply:
Yeah, either that or what is currently here. I don't have a strong preference either way. The official Bazel docs mention Homebrew as one of the options.